plugins/katello/developers.md

```diff
@@ -255,41 +190,25 @@ Now, lock down Katello versions to the appropriate minor, e.g. if you release `p
 mod 'katello',   '>= 0.2.0 < 0.3.0'
 ```
 
-### Step 6: Configure Koji
+### Step 7: Configure Koji
 
-Note that to perform this step, Foreman must have generated it's Koji build targets for the targeted Foreman release. This is due to how Katello tags inherit from some Foreman tags. Coordinate with the Foreman release nanny on when they plan to do that step and wait to do this step until then. Koji needs to have configuration added specific to the tags that the release will occur under. The first step is to add new build roots and tags using the following script:
+Note that to perform this step, Foreman must have generated it's Koji build targets for the targeted Foreman release. This is due to how Katello tags inherit from some Foreman tags. Coordinate with the Foreman release nanny on when they plan to do that step and wait to do this step until then. Koji needs to have configuration added specific to the tags that the release will occur under. The first step is to add new build roots and tags:
 
 ```
-https://github.com/Katello/katello-misc/blob/master/rel-eng/koji_create_buildroots.sh
+./tools koji configs/katello_XY.yaml # Make sure it looks good
+./tools koji --commit configs/katello_XY.yaml
```

Comment:
Maybe add a note above or below that this step requires Koji access and that if the release nanny does not have access to contact the `-dev` list to request it.